pkg/registry/apis/provisioning/webhooks/webhook.go

```diff
@@ -67,6 +73,32 @@ func (*webhookConnector) NewConnectOptions() (runtime.Object, bool, string) {
 	return nil, false, ""
 }
 
+func (s *webhookConnector) Authorize(ctx context.Context, a authorizer.Attributes) (decision authorizer.Decision, reason string, err error) {
```

Comment:
Notice how now we have more self-contained logic for the endpoint with authorization, open API spec and routes.